pyproject.toml

```diff
@@ -1,5 +1,8 @@
 [build-system]
-requires = ["setuptools>=45", "setuptools_scm[toml]>=6.2"]
+requires = [
+    "setuptools>=45",
+    "setuptools_scm[toml]>=6.2"
```

Comment:
maybe end with ``,`` to make it diff-easy

Reply:
same goes for everywhere else

Reply:
done